src/Domain/AuthorisationService.cs

```diff
@@ -1,31 +1,31 @@
-﻿namespace Linn.Products.Domain
+namespace Linn.Products.Domain
 {
     using System.Collections.Generic;
     using System.Linq;
 
     public class AuthorisationService : IAuthorisationService
     {
-        public bool HasPermissionFor(AuthorisedAction action, IEnumerable<string> privileges)
+        public bool HasPermissionFor(string action, IEnumerable<string> privileges)
         {
             switch (action)
             {
```

Comment:
In fact do you even need the switch anymore? Just call the satisfies method regardless of what they're trying to do.

Reply:
👍